collections/_posts/2019-08-29-adding-custom-doh-resolvers-to-dnscloak.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Adding Custom DNS over HTTPS Resolvers to DNSCloak
+author: nitrohorse
+editor: Jonah
+layout: post
+tags:
+  - guides
+  - dns
+  - dnscrypt
+  - ios
+---
+
+**[DNSCloak](https://apps.apple.com/us/app/dnscloak-secure-dns-client/id1452162351)** is an [open-source](https://github.com/s-s/dnscloak) DNSCrypt and DNS over HTTPS (DoH) client for iOS, which gives users the ability to encrypt their DNS requests through the use of an on-device VPN profile.
+
+While highly configurable, its user interface can be unintuitive and doesn't easily allow users to add custom DoH resolvers apart from the default ["public-resolvers" list](https://github.com/DNSCrypt/dnscrypt-resolvers/blob/master/v2/public-resolvers.md) that the DNSCrypt project provides.
+
+This guide will walk you through setting up DNSCloak to connect to any public resolver that supports either DNSCrypt or DoH.
+
+While there is a lot of nuance to DNSCrypt and DoH, it's important to understand that these two DNS protocols essentially achieve the same goals: provide users with the ability to encrypt all DNS traffic to the users' desired [upstream provider(s)](https://www.privacytools.io/providers/dns/#icanndns), while preventing [DNS hijacking](https://en.wikipedia.org/wiki/DNS_hijacking), [spoofing](https://en.wikipedia.org/wiki/DNS_spoofing), and eavesdropping by 3rd parties.
```

Comment:
This doesn't seem to fit well here. Nor the paragraph below. Will need to think through this more.